jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -35,22 +35,15 @@
     "vue-upload-component": "^3.1.2"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.19.1",
-    "@babel/register": "^7.18.9",
     "@cypress/code-coverage": "^3.10.0",
     "@cypress/vue": "^4.2.0",
     "@cypress/webpack-preprocessor": "^5.12.2",
     "@istanbuljs/nyc-config-babel": "^3.0.0",
-    "@vue/cli-plugin-babel": "~5.0.8",
-    "@vue/cli-plugin-e2e-cypress": "^5.0.8",
-    "@vue/cli-plugin-eslint": "~5.0.8",
-    "@vue/cli-plugin-unit-mocha": "~5.0.8",
-    "@vue/cli-service": "~5.0.8",
+    "@vitejs/plugin-vue": "^4.0.0",
```

Comment:
Less dependencies as Vite does many of what these other tools do (and has been adopted by devs doing Vue, React, Angular, ThreeJS, so now it's stable and future-prone too :crossed_fingers: )